### PR TITLE
3014 improvement common  scrolling zones on pages

### PR DESCRIPTION
**Pull request by CREDO23**
## Description

What I did optimize the spacing in home and profile pages

## Type of Change

-   [ ] Bug fix
-   [ ] New feature
-   [ ] Breaking change
-   [ ] Documentation update

## Checklist

-   [x] My code follows the style guidelines of this project
-   [ ] I have performed a self-review of my code
-   [ ] I have commented my code, particularly in hard-to-understand areas
-   [ ] I have made corresponding changes to the documentation
-   [x] My changes generate no new warnings

### CODE REVIEW

File: apps/web/lib/features/team/user-team-card/task-skeleton.tsx [hunk 1]
```diff
@@ -61,15 +61,15 @@ export function UserTeamCardHeader() {
 					<div className="w-4 self-stretch border-l-[0.125rem] border-l-transparent " />
 					<div className="2xl:w-48 3xl:w-[12rem] w-1/5 lg:px-4 !pl-6 lg:!pl-8  flex flex-col items-center text-center  justify-center">
 						<Tooltip label={t('task.taskTableHead.WORKED_ON_TASK_HEADER_TOOLTIP')}>
-							{t('task.taskTableHead.TASK_WORK.TITLE')} <br /> {t('common.TASK')}
+							{t('common.TASK')} Time
```
Comment (evereq) [inline apps/web/lib/features/team/user-team-card/task-skeleton.tsx:64]: @CREDO23 hm, why we are doing this change? Now "Time" become hardcoded and will NOT be translated!

File: apps/web/lib/features/team/user-team-card/task-skeleton.tsx [hunk 2]
```diff
@@ -61,15 +61,15 @@ export function UserTeamCardHeader() {
...
 						</Tooltip>
 					</div>
 					<div className="w-1 self-stretch border-l-[0.125rem] border-l-transparent" />
 					<div className="w-1/5 lg:px-3 2xl:w-52 3xl:w-64 !pl-14 text-center">{t('common.ESTIMATE')}</div>
 					<div className="w-1 self-stretch border-l-[0.125rem] border-l-transparent" />
 					<div className="flex justify-center items-center cursor-pointer w-1/5 gap-4 lg:px-3 2xl:w-52 max-w-[13rem] !pl-14 text-center">
 						<Tooltip label={t('task.taskTableHead.TOTAL_WORKED_TODAY_HEADER_TOOLTIP')}>
-							{t('task.taskTableHead.TOTAL_WORK.TITLE')} <br /> {t('common.TODAY')}
+							Total {t('common.TODAY')}
```
Comment (evereq) [inline apps/web/lib/features/team/user-team-card/task-skeleton.tsx:72]: @CREDO23 hm, why we are doing this change? Now "Total" become hardcoded and will NOT be translated!

**Comment by evereq** [comment]
@CREDO23 please see my comments. Also it seems your formatting is different to existed files so a LOT of code formatting changing that make it hard to review PR...

**Comment by CREDO23** [comment]
> @CREDO23 please see my comments. Also it seems your formatting is different to existed files so a LOT of code formatting changing that make it hard to review PR...

yes I see

File: apps/web/lib/features/team/user-team-card/task-skeleton.tsx (hunk 1 above)
Comment (CREDO23) [reply apps/web/lib/features/team/user-team-card/task-skeleton.tsx:64]: Let me translate

**Pull request merged**